src/main.ts

```diff
@@ -66,6 +67,7 @@ Vue.directive('click-outside', clickOutsideDirective);
 /// end-region directives
 
 const app = new Vue({
+    // @ts-ignore
```

Comment:
vue-router causes some conflicts with typescript version which produce console errors sometimes, here is a reference https://github.com/vuejs/vue-router/issues/3019

Reply:
Interesting that it happens sometimes, perhaps that's the reason, I couldn't see any errors in my local console, could you provide any logs regarding this issue if possible? Keen to understand the issue better, that'd be nice if we can find a better solution rather than suppressing it.

Reply:
<img width="813" alt="Screen Shot 2021-08-24 at 2 28 06 PM" src="https://user-images.githubusercontent.com/9143194/130616460-10594e5d-3dfc-438e-a1cf-76799ad273e2.png">